ubuntu22.04/precompiled/Dockerfile

```
@@ -65,7 +65,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends nvidia-driver-$
 # avoid cleaning the cache after this to retain these packages during runtime.
 RUN apt-get update && apt-get install --download-only --no-install-recommends -y linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
     linux-signatures-nvidia-${KERNEL_VERSION} \
-    linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}
+    linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
+    # add support for nvidia open source driver packages during runtime
+    linux-objects-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION} \
+    linux-modules-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION}
```

As discussed over slack, we should only need the `linux-modules-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION}` package.
```suggestion
    # add support for nvidia open source driver packages during runtime
    linux-modules-nvidia-${DRIVER_BRANCH}-server-open-${KERNEL_VERSION}
```